debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/type/AbstractType.java

```diff
@@ -54,9 +54,9 @@ public String getDefaultValueBinding(DatabaseDialect dialect, Schema schema, Obj
             case FLOAT64:
                 return Double.toString((double) value);
             case STRING:
-                return "'" + ((String) value).replaceAll("(?<!')'(?!')", "''") + "'";
+                return "'" + value + "'";
```

Comment:
```suggestion
                return "'" + ((String) value).replaceAll("(?<!')'(?!')", "''") + "'";
```
This needs to use the old behavior, otherwise default values that contain `'` characters will be mishandled. This was a recent change that may have gotten lost in your fork.

Reply:
Yep. Merged from the other dev branch incorrectly.